polyaxon/one_gpu_classification.yml

```diff
@@ -45,4 +42,4 @@ params:
 
 run:
   cmd:
-    - python src/bilstm_train_classification.py --input_path {{ input_path }} --train_filename {{ train_filename }} --test_filename {{ test_filename }} --models_path {{ models_path }} --tokenizer_file {{ tokenizer_file }} --embedding_path {{ embedding_path }} --embedding_file {{ embedding_file }} --embedding_matrix_file {{ embedding_matrix_file }} --weights_file {{ weights_file }} --max_features {{ max_features }} --max_length {{ max_length }}  --embed_size {{ embed_size }}  --epochs {{ epochs }}  --batch_size {{ batch_size }}  --learning_rate {{ learning_rate }} --patience {{ patience }}
\ No newline at end of file
+    - python src/bilstm_train_classification.py  --models_path {{ models_path }} --tokenizer_file {{ tokenizer_file }} --embedding_path {{ embedding_path }} --embedding_file {{ embedding_file }} --embedding_matrix_file {{ embedding_matrix_file }} --weights_file {{ weights_file }} --max_features {{ max_features }} --max_length {{ max_length }}  --embed_size {{ embed_size }}  --epochs {{ epochs }}  --batch_size {{ batch_size }}  --learning_rate {{ learning_rate }} --patience {{ patience }}
```

Comment:
I meant: `python src/bilstm_train_classification.py --input_path /data/serendeepia/prolexitim_alexitimia/20190920_prolexytim_analytics ...` instead of `python src/bilstm_train_classification.py --input_path {{ input_path }}`